PowerPointLabs/PowerPointLabs/ActionFramework/Action/FitToHeightActionHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
```

Comment:
Idea: create a action handler for your feature (a subclass of ActionHandler)
and provide the ribbon id that it should work for using ExportActionRibbonIdAttribute.